packages/@webex/plugin-meetings/src/reachability/clusterReachability.ts

```diff
@@ -70,7 +70,7 @@ export class ClusterReachability {
    * @returns {Number} Milliseconds
    */
   private getElapsedTime() {
-    return Math.round(performance.now() - this.startTimestamp);
+    return Math.round(performance.now() - (this.startTimestamp || 0));
```

Comment:
`this.startTimestamp` wasn't initiated, so the type tells me that it can be undefined. I put a fallback to make types not complain about that.

Reply:
this function should probably throw if it's called before `this.startTimestamp` is set, as this should never happen